cfme/fixtures/v2v_fixtures.py

```diff
@@ -562,6 +568,10 @@ def mapping_data_single_network(request, source_provider, provider):
 
 @pytest.fixture(scope="function")
 def mapping_data_dual_vm_obj_dual_datastore(request, appliance, source_provider, provider):
+    # Picking two datastores on Vmware and Target provider for this test
+    source_type = ['nfs', 'iscsi']
+    dest_type = ['nfs', 'iscsi']
```

Comment:
_optional_ 

```suggestion
    dest_type = source_type
```

Reply:
Can even one-line it, equating all three.